lib/airbrakex/notifier.ex

```diff
@@ -61,6 +61,13 @@ defmodule Airbrakex.Notifier do
     "#{endpoint}/api/v3/projects/#{project_id}/notices?key=#{project_key}"
   end
 
+  defp http_options do
+    case Config.get(:airbrakex, :ssl) do
+      nil -> []
+      ssl_config -> [ssl: ssl_config]
```

Comment:
Actually thinking about it, I've not checked but what other options can be passed to post? Are there are other options? Can we pass an arbitrary configuration block?

Reply:
As far as I know, there is no such thing as "block" in Elixir.

@danmarcab can we use `Config.get(:airbrakex, :ssl) || []` here instead of `case` statement?

Reply:
From HTTPoison docs:

```
Options:

  • :timeout - timeout to establish a connection, in milliseconds. Default
    is 8000
  • :recv_timeout - timeout used when receiving a connection. Default is
    5000
  • :stream_to - a PID to stream the response to
  • :async - if given :once, will only stream one message at a time,
    requires call to stream_next
  • :proxy - a proxy to be used for the request; it can be a regular url or
    a {Host, Port} tuple, or a `{:socks5, ProxyHost, ProxyPort} tuple
  • :proxy_auth - proxy authentication {User, Password} tuple
  • :socks5_user- socks5 username
  • :socks5_pass- socks5 password
  • :ssl - SSL options supported by the ssl erlang module
  • :follow_redirect - a boolean that causes redirects to be followed
  • :max_redirect - an integer denoting the maximum number of redirects to
    follow
  • :params - an enumerable consisting of two-item tuples that will be
    appended to the url as query string parameters
```
We could name the config option `http_options` or something, but I was not sure we want to expose all. Happy to hear suggestions

Reply:
I think it would only make sense to expose 
```
  • :timeout - timeout to establish a connection, in milliseconds. Default
    is 8000
  • :recv_timeout - timeout used when receiving a connection. Default is
    5000
  • :proxy - a proxy to be used for the request; it can be a regular url or
    a {Host, Port} tuple, or a `{:socks5, ProxyHost, ProxyPort} tuple
  • :proxy_auth - proxy authentication {User, Password} tuple
  • :socks5_user- socks5 username
  • :socks5_pass- socks5 password
  • :ssl - SSL options supported by the ssl erlang module
```

Reply:
I'm purely thinking of this for upstream submission - the better the feature the more likely it is to be accepted ;)